vendor/QRCode/QRMath.js

```
@@ -0,0 +1,44 @@
+	},
+	
+	EXP_TABLE : new Array(256),
+	
+	LOG_TABLE : new Array(256)
+
+};
+	
+for (var i = 0; i < 8; i++) {
+	QRMath.EXP_TABLE[i] = 1 << i;
+}
+for (var i = 8; i < 256; i++) {
```

Biome (2.1.2)

[error] Shouldn't redeclare 'i'. Consider to delete it or rename it.

'i' is defined here:


(lint/suspicious/noRedeclare)